src/work-allocation-2/containers/work-case-list/work-case-list.component.scss

```diff
@@ -0,0 +1,119 @@
+.actions {
+  display: flex;
+  justify-content: flex-end;
+
+  .action {
+    padding: 2px;
+  }
+}
+/**
+ * Used to override the border-bottom style being set by the GDS .govuk-table__cell
+ */
+.border-none {
+  border: 0px;
+}
+
+.reset-sort-button {
+  float: right;
+  margin-bottom: -10px;
+  margin-top: -10px;
+}
+
+.column-sort {
+  padding-right: 20px;
+
+  &:hover{
```

Comment:
_Nitpick:_
Stylistically, not seeing the space before the `{` is aesthetically displeasing. The same as `&:after` and `&:before`.

Reply:
I've formatted the entire file now and this has been changed.